src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        String[] figures = {"Circle", "Square", "Rectangle", "RightTriangle", "IsoscelesTrapezoid"};
```

Comment:
imagine you'll get another figure to draw. you'll need to add switch and add a value to array. can we choose what figure to create based on a int number?